app/javascript/flavours/glitch/locales/en.json

```diff
@@ -103,6 +103,7 @@
   "settings.auto_collapse_all": "Everything",
   "settings.auto_collapse_lengthy": "Lengthy toots",
   "settings.auto_collapse_media": "Toots with media",
+  "settings.auto_collapse_height": "Height (pixels) of lengthy toots",
```

Comment:
I think this is unclear.
I actually prefer the wording in the earlier PRs: "Maximum height (pixels) of toots before auto-collapsing",
or "Height (in pixels) to consider a toot lengthy" (or similar).

Reply:
I like the latter suggestion. The previous language was unclear as to it's relationship to the "lengthy toots" option so your suggestion works great